src/module/rules/rule-element/battle-form/schema.ts

```diff
@@ -27,7 +35,13 @@ type BattleFormRuleOverrideSchema = {
         false
     >;
     tempHP: ResolvableValueField<false, true, true>;
-    senses: ObjectField<BattleFormSenses, BattleFormSenses, false, false, false>;
+    senses: RecordField<
+        StringField<string, string, true, false, false>,
```

Comment:
This should then be the set of of the permitted senses rather than a mere `string`

Reply:
Done. A normal `Record` in `RecordFieldModelProp` was enough after all.

Reply:
ok it had to be a partial `Record` after all.

Reply:
Right, these will globally be `Record<string, something | undefined>`